nemo_rl/experience/rollouts.py

```diff
@@ -120,7 +120,7 @@ def calculate_rewards(
     """
     # Extract message logs for environment (most recent interaction)
     to_env = [
-        get_keys_from_message_log(batch["message_log"][i], ["role", "content"])
+        get_keys_from_message_log(batch["message_log"][i], ["role", "content", "content_in_oai_format"])
```

Comment:
is this part of the logic necessary? I'd like to avoid having environments modify the rollout loop.